test/NuGet.Core.Tests/NuGet.Configuration.Test/FallbackFolderTests.cs

```diff
@@ -185,7 +185,7 @@ public void GetFallbackPackageFolders_MultipleFoldersFromMultipleNuGetConfigs()
                     .GetFallbackPackageFolders(settings)
                     .Select(GetFileName);
 
-                var expected = new[] { "a", "b", "c", "d", "x", "y" };
+                var expected = new[] { "x", "y", "c", "d", "a", "b" };
```

Comment:
Is this change correct?  Would returning the fallback folders in a different order affect package resolution?